src/Preferences.m

```diff
@@ -272,7 +270,9 @@ -(NSDictionary *)allocFactoryDefaults
 	defaultValues[MAPref_FilterMode] = @MA_Filter_All;
 	defaultValues[MAPref_MinimumFontSize] = @(MA_Default_MinimumFontSize);
 	defaultValues[MAPref_AutoExpireDuration] = @(MA_Default_AutoExpireDuration);
-	defaultValues[MAPref_DownloadsFolder] = MA_DefaultDownloadsFolder;
+    defaultValues[MAPref_DownloadsFolder] = [NSFileManager.defaultManager
+                                             URLsForDirectory:NSDesktopDirectory
+                                             inDomains:NSUserDomainMask].firstObject.absoluteString;
```

Comment:
Use `path` here instead of `absoluteString`. Path resolves to a Unix file path, whereas absoluteString is a string representation of a URL. There are just two entities that use this path, which can both work with this.